server/src/test/java/org/elasticsearch/action/admin/indices/alias/GetAliasesResponseTests.java

```diff
@@ -242,4 +248,32 @@ public void testFromXContentWithNoAliasFound() throws IOException {
         }
     }
 
+    public void testSerializationBwc() throws IOException {
```

Comment:
this test is good, it tests that the updated serialization works well with the different versions. Another one that we could have would read from a static base64 of this response generated from a previous branch, to really make sure that the current code can read what the old code outputs. I can also add that as a followup if you don't have time.

Reply:
@javanna can you have a look if this is what you meant? 6.3 has been used to serialize the response.

Reply:
looks great, thanks!